<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pile Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="appres.qrc">
    <normaloff>:/icons/resources/icons/package.png</normaloff>:/icons/resources/icons/package.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 197);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTreeView" name="treeView">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 127);</string>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <property name="animated">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Commands</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Explore</string>
         </property>
         <property name="icon">
          <iconset resource="appres.qrc">
           <normaloff>:/icons/resources/icons/folder_explore.png</normaloff>:/icons/resources/icons/folder_explore.png</iconset>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="btn_add_pile">
         <property name="text">
          <string>Add</string>
         </property>
         <property name="icon">
          <iconset resource="appres.qrc">
           <normaloff>:/icons/resources/icons/package_add.png</normaloff>:/icons/resources/icons/package_add.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btn_open">
         <property name="text">
          <string>Open</string>
         </property>
         <property name="icon">
          <iconset resource="appres.qrc">
           <normaloff>:/icons/resources/icons/rocket.png</normaloff>:/icons/resources/icons/rocket.png</iconset>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="btn_new_pile">
         <property name="text">
          <string>New</string>
         </property>
         <property name="icon">
          <iconset resource="appres.qrc">
           <normaloff>:/icons/resources/icons/package.png</normaloff>:/icons/resources/icons/package.png</iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="btn_settings">
         <property name="text">
          <string>Settings</string>
         </property>
         <property name="icon">
          <iconset resource="appres.qrc">
           <normaloff>:/icons/resources/icons/setting_tools.png</normaloff>:/icons/resources/icons/setting_tools.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>564</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBack"/>
   <addaction name="actionForward"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="actionCopy_and_replace"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionSet_root_here">
   <property name="icon">
    <iconset resource="appres.qrc">
     <normaloff>:/icons/resources/icons/application_get.png</normaloff>:/icons/resources/icons/application_get.png</iconset>
   </property>
   <property name="text">
    <string>Set root here</string>
   </property>
   <property name="toolTip">
    <string>Sets a virtual root to selected directory</string>
   </property>
  </action>
  <action name="actionClear_virtual_root">
   <property name="icon">
    <iconset resource="appres.qrc">
     <normaloff>:/icons/resources/icons/application_home.png</normaloff>:/icons/resources/icons/application_home.png</iconset>
   </property>
   <property name="text">
    <string>Clear virtual root</string>
   </property>
   <property name="toolTip">
    <string>Sets the root of the tree to top level directory</string>
   </property>
  </action>
  <action name="actionAdd_to_favorites">
   <property name="icon">
    <iconset resource="appres.qrc">
     <normaloff>:/icons/resources/icons/star.png</normaloff>:/icons/resources/icons/star.png</iconset>
   </property>
   <property name="text">
    <string>Add to favorites</string>
   </property>
   <property name="toolTip">
    <string>Adds current item to favorites</string>
   </property>
  </action>
  <action name="actionGo_to_path">
   <property name="icon">
    <iconset resource="appres.qrc">
     <normaloff>:/icons/resources/icons/dialog.png</normaloff>:/icons/resources/icons/dialog.png</iconset>
   </property>
   <property name="text">
    <string>Go to path ...</string>
   </property>
   <property name="toolTip">
    <string>Allows you to enter a path and selects it</string>
   </property>
  </action>
  <action name="actionGo_to_clipboard_path">
   <property name="icon">
    <iconset resource="appres.qrc">
     <normaloff>:/icons/resources/icons/clipboard_invoice.png</normaloff>:/icons/resources/icons/clipboard_invoice.png</iconset>
   </property>
   <property name="text">
    <string>Go to clipboard path</string>
   </property>
   <property name="toolTip">
    <string>Gets the text in the clipboard and interprets it as a path</string>
   </property>
  </action>
  <action name="actionRemove_all_favorites">
   <property name="icon">
    <iconset resource="appres.qrc">
     <normaloff>:/icons/resources/icons/cross.png</normaloff>:/icons/resources/icons/cross.png</iconset>
   </property>
   <property name="text">
    <string>Remove all favorites</string>
   </property>
   <property name="toolTip">
    <string>Removes all saved favourites</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="appres.qrc">
     <normaloff>:/icons/resources/icons/control_play_blue_left.png</normaloff>:/icons/resources/icons/control_play_blue_left.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="toolTip">
    <string>Navigate to previous location</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset resource="appres.qrc">
     <normaloff>:/icons/resources/icons/control_play_blue.png</normaloff>:/icons/resources/icons/control_play_blue.png</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="toolTip">
    <string>Navigate to next location</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="appres.qrc">
     <normaloff>:/icons/resources/icons/page_copy.png</normaloff>:/icons/resources/icons/page_copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy selected items</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="appres.qrc">
     <normaloff>:/icons/resources/icons/cut_red.png</normaloff>:/icons/resources/icons/cut_red.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut selected items</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="appres.qrc">
     <normaloff>:/icons/resources/icons/page_paste.png</normaloff>:/icons/resources/icons/page_paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste items</string>
   </property>
  </action>
  <action name="actionAdd_custom_command">
   <property name="icon">
    <iconset resource="appres.qrc">
     <normaloff>:/icons/resources/icons/add.png</normaloff>:/icons/resources/icons/add.png</iconset>
   </property>
   <property name="text">
    <string>Add custom command ...</string>
   </property>
   <property name="toolTip">
    <string>Allows defining commands to be run in this menu</string>
   </property>
  </action>
  <action name="actionCopy_and_replace">
   <property name="icon">
    <iconset resource="appres.qrc">
     <normaloff>:/icons/resources/icons/calendar_copy.png</normaloff>:/icons/resources/icons/calendar_copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy and replace</string>
   </property>
   <property name="toolTip">
    <string>Copy the files and replace the content</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="appres.qrc"/>
 </resources>
 <connections/>
</ui>
